android/app/src/main/assets/microlocations.json

```diff
@@ -1,138 +1,138 @@
 [
     {
-        "floor": null, 
-        "id": 1, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 3, Level 3, Dalton Hall", 
+        "floor": null,
+        "id": 1,
+        "latitude": null,
+        "longitude": null,
+        "name": "Level 3, Level 3, Dalton Hall",
         "room": null
-    }, 
-    {
-        "floor": null, 
-        "id": 2, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 3, Lewis Lab", 
+    },
+    {
+        "floor": null,
+        "id": 2,
+        "latitude": null,
+        "longitude": null,
+        "name": "Level 3, Lewis Lab",
         "room": null
-    }, 
-    {
-        "floor": null, 
-        "id": 3, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 1, Digital Design Studio", 
+    },
+    {
+        "floor": null,
+        "id": 3,
+        "latitude": null,
+        "longitude": null,
+        "name": "Level 1, Digital Design Studio",
         "room": null
-    }, 
-    {
-        "floor": null, 
-        "id": 4, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 1, Digital Design Studio", 
+    },
+    {
+        "floor": null,
+        "id": 4,
+        "latitude": null,
```

Comment:
@mananwason the geopoints are not visible in the indoor map of the location. What I have found is an image of the indoor map on their offical website. Do I need to manually locate each indoor location or is there a better way to do it?